### ISSUE TITLE
Error --no-duplicates arg

### ISSUE THREAD

**Issue by Midaxess** [marevers/pleasant-cli#1]
Hello,

I have an issue when I use the argument "--no-duplicates" in a entry or folder creation

An exemple of what I did :

```
$ pleasant-cli create entry --no-duplicates --data ' 
{
    "Username": "test",
    "Password": "password",
    "GroupId": "xxxxxxxxx"
}' 
```

And I have this answer :

`json: cannot unmarshal object into Go struct field Folder.Children.Credentials of type string`

When I remove the argument, the command works

I use the last version 0.0.2 on a Linux machine

I already try to run it on other Linux machine but same problem

Thx for you help

**Comment by marevers** [comment]
@Midaxess thank you for opening an issue.

Your entry requires a name, else --no-duplicates will not be able to look for an entry with the same name to see if it already exists.

A valid JSON structure for this command would be:

```
$ pleasant-cli create entry --no-duplicates --data ' 
{
    "Name": "TestEntry",
    "Username": "test",
    "Password": "password",
    "GroupId": "xxxxxxxxx"
}' 
```

Upon initial creation (assuming TestEntry does not already exist), creation will succeed. If the command is repeated, you'll receive `error: duplicate entry found, skipping creation`.

Please let me know if that helped!

**Comment by Midaxess** [comment]
Thanks for your answer,

I did your command :
```
pleasant-cli create entry --no-duplicates --data ' 
{
    "Name": "TestEntry",
    "Username": "test",
    "Password": "password",
    "GroupId": "xxxxxxxxx"
}' 
```
But same issue.

When I run it when my entry doesn't exist and I always have this :
`json: cannot unmarshal object into Go struct field Folder.Children.Credentials of type string
`

Same when I create the entry without "--no-duplicates" and then add it to check whether it exists.

I tried these commands on 2 different Linux machine.

**Comment by marevers** [comment]
Interesting. I am unable to reproduce this error on my system. Probably I'll need some more information:

- What version of Pleasant Password Server are you using? Does that support API version v5?
- Are you using the correct version of pleasant-cli for your system?
- Can you try executing a `pleasant-cli get folder --id <id>` for the folder the entry is in? Does that work? If so, it would help to have an idea of the output of that command. Feel free to redact any sensitive information from it.

**Comment by Midaxess** [comment]
- It seems that the server version is 7.11.14.0
- I'm on Debian 11 / 12 so I use pleasant-cli_0.0.2_linux_x86_64.tar.gz
- Yes it works

I've just realised that I only get the error when I try to create an entry in a folder that contains subfolders.

For example: I just have the folder test-pleasant-cli which contains nothing. If I try to create an entry, no problem. However, if I create a folder in this one, for example sub-test-pleasant-cli, and I try to add an entry in test-pleasant-cli, I get an error.

Let me know if you still need output of my command

**Comment by marevers** [comment]
Thanks. I just tested that exact case - creating a subfolder in 'folder-a' called 'folder-b', and then in 'folder-a' create an entry with the subfolder being present. At that point, the error still does not occur. However, once I add an entry into 'folder-b' **and then** try to create an entry in parent 'folder-a', I get the same error.

I need to investigate where it's coming from but at least I can reproduce it now. I'll provide an update once I fix the root cause of the error.

**Comment by marevers** [comment]
@Midaxess I have just released [v0.0.3](https://github.com/marevers/pleasant-cli/releases/tag/v0.0.3) which should fix the issue you are encountering. Please test with that version and let me know if it works!

**Comment by Midaxess** [comment]
Works like a charm !

Thank you for your responsiveness.
